teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] in the initialize method of this class `age`, `name` and `parent_permission` should be passed as arguments as well,  please remember that `name` and `parent_permission` should both have a default value as well.